server/mediaserver.go

```diff
@@ -161,15 +178,13 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 			minDeposit := big.NewInt(0).Mul(BroadcastPrice, big.NewInt(MinDepositSegmentCount))
 			if deposit.Cmp(minDeposit) < 0 {
 				glog.Errorf("Low deposit (%v) - cannot start broadcast session", deposit)
+				if s.LivepeerNode.MonitorMetrics {
+					monitor.LogStreamCreateFailed(rtmpStrm.GetStreamID(), 0, "LowDeposit")
```

Comment:
Is there a reason for using `0` instead of `nonce`?

Reply:
I was thinking 0 means the stream failed before it was created... But I don't like that anymore.  Will change it to using nonce.